src/routes/help/submitting/+page.svelte

```diff
@@ -173,9 +175,9 @@
 </div>
 
 <h2>JSON Snippet</h2>
-<pre>
-	{JSON.stringify(jsonSnippet,null,4)}<button on:click={copy}>{clipboardCopy ? 'Copied' : 'Copy'}</button>
-</pre>
+<pre>{JSON.stringify(jsonSnippet, null, 4).replaceAll('    ', '\t')}<button on:click={copy}
```

Comment:
I think you could write it as:
```suggestion
<pre>{JSON.stringify(jsonSnippet, null, '\t')}<button on:click={copy}
```

Reply:
Oh, I didn't know that the third parameter can be a string